.vscode/settings.json

```diff
@@ -30,6 +30,7 @@
     },
     "[jsonc]": {
       "editor.defaultFormatter": "esbenp.prettier-vscode"
-    }
+   },
+   "editor.tabSize": 2
```

Comment:
is there a reason for changing this config? make sure your editor is setup properly if things are formatting weird. I can help you out with that if you need

Reply:
Ah yeah looks like this was an accident! Will remove